Components/List/list.css

```diff
@@ -0,0 +1,77 @@
+img {
```

Comment:
Instead of selecting the element you can use a class instead related to it and pass these properties and use this same class in the html where you intend to?